common/src/main/java/com/wynntils/core/components/Services.java

```diff
@@ -47,5 +49,8 @@ public final class Services {
     public static final StopwatchService Stopwatch = new StopwatchService();
     public static final TranslationService Translation = new TranslationService();
     public static final UpdateService Update = new UpdateService();
+    public static final WaypointsService Waypoints = new WaypointsService();
     public static final WynntilsAccountService WynntilsAccount = new WynntilsAccountService();
+
+    public static final UserMarkerService UserMarker = new UserMarkerService(MapData);
```

Comment:
Kind of sad that we need to introduce a Service with dependencies on another Service. :(

Reply:
It's not longer needed anymore.